man/systemd-repart.xml

```diff
@@ -363,6 +363,18 @@
         if <option>--split</option> is enabled.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--include-partitions=</option><arg rep="repeat">PARTITION</arg></term>
+        <term><option>--exclude-partitions=</option><arg rep="repeat">PARTITION</arg></term>
+
+        <listitem><para>These options specify which partition types <command>systemd-repart</command> should
+        operate on. Any partitions of types that are filtered by these options are still taken into account
+        when calculating the sizes and offsets of other partitions, but aren't actually written to the disk
+        image. The net effect of this option is that if you run <command>systemd-repart</command> again
+        without these options, the missing partitions will be added as if they had not been excluded the
+        first time <command>systemd-repart</command> was executed.</para></listitem>
```

Comment:
Would be good for extra clarity to say the lists are comma separated, and that "include" means only those will be operated upon (like --help says)